src/common/query/src/logical_plan/udf.rs

```diff
@@ -68,25 +71,61 @@ impl ScalarUdf {
     }
 }
 
+#[derive(Clone)]
+struct DfUdfAdapter {
+    name: String,
+    signature: datafusion_expr::Signature,
+    return_type: datafusion_expr::ReturnTypeFunction,
+    fun: DfScalarFunctionImplementation,
+}
+
+impl Debug for DfUdfAdapter {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        f.debug_struct("DfUdfAdapter")
+            .field("name", &self.name)
+            .field("signature", &self.signature)
+            .field("fun", &"<FUNC>")
```

Comment:
sugg: keep it aligned with `DfUdafAdapter`?

```suggestion
            .field("fun", &"<FUNC>")
```

Reply:
Updated.